saas/demo/saas_template_demo.xml

```diff
@@ -0,0 +1,16 @@
+<!-- Copyright 2019 Denis Mudarisov <https://it-projects.info/team/trojikman>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).-->
+<odoo>
+    <record id="saas_template" model="saas.template">
+        <field name="name">Template</field>
+        <field name="template_module_ids" eval="[(4, ref('mail')),]"/>
+        <field name="template_demo" eval="True"/>
+        <field name="template_post_init">env['res.partner'].create({'name': 'Name'})</field>
```

Comment:
Use more informative name.
"Example of partner creation on template initialization"